packages/eip-1193-provider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@turnkey/eip-1193-provider",
-  "version": "3.0.0",
+  "version": "4.0.0",
```

Comment:
Where did this come from? This is a major change. And I only see "patch changes" in the corresponding CHANGELOG. Something isn't right.

Reply:
we've also moved from 2 -> 3 on a patch change in the past, should we mess with the autogen?